src/util/aquarius.ts

```diff
@@ -69,13 +84,34 @@ export function getSearchQuery(
   const filters: FilterTerms[] = [
     ...(baseQuery.query.bool?.filter as FilterTerms[])
   ]
+  const sortKey = sortBy
+    ? defaultSortByFields[sortBy]
+    : defaultSortByFields[SortByOptions.Relevance]
 
   if (type)
+    filters.push(
+      type === 'all'
```

Comment:
suggstion(blocking): `type = 'all'` could be the default (`type` != `MetdataMain[type]`), otherwise add `'all'` on this functions level to the `MetdataMain[type]` types:
```
type?: MetadataMain['type'] | 'all'
```
This should clean up the code here a bit. We only need the `term` query if a type is specified.